### PR TITLE
Make the primary, flow-advancing action the right-most

### PR DESCRIPTION
**Pull request by tacotexmex**
This should be a no-brainer. UX standards dictates that the primary action should be put to the right.

### CODE REVIEW

**Comment by paramat** [comment]
I support this, but the button co-ordinates have not been changed?

File: builtin/mainmenu/tab_local.lua [hunk 1]
```diff
@@ -94,8 +94,8 @@ local function get_formspec(tabview, name, tabdata)
 
 	retval = retval ..
 			"button[4,3.95;2.6,1;world_delete;".. fgettext("Delete") .. "]" ..
-			"button[6.5,3.95.15;2.8,1;world_create;".. fgettext("New") .. "]" ..
 			"button[9.2,3.95;2.5,1;world_configure;".. fgettext("Configure") .. "]" ..
+			"button[6.5,3.95.15;2.8,1;world_create;".. fgettext("New") .. "]" ..
```
[the remark this answers is not in the document]
Comment (paramat) [reply builtin/mainmenu/tab_local.lua]: ^ That.

**Comment by tacotexmex** [comment]
Holy mese I’m stupid. xD Let me correct that later.

**Comment by SmallJoker** [comment]
Indeed, that's a little too much of a "no-brainer". Although, the concept is fine.

**Comment by paramat** [comment]
Did you test it? Screenshot?

**Comment by ClobberXD** [comment]
Haven't tested, but the code seems alright to me. Now the button order will _really_ be changed... :)

https://github.com/minetest/minetest/blob/bf45644041693f959ac95154b00f15aa27b34145/builtin/mainmenu/tab_local.lua#L97

`3.95.15`? Is this intended? If not, @tacotexmex could change it to `3.95`

**Comment by tacotexmex** [comment]
Tested and working.

**Comment by paramat** [comment]
Well spotted, that '.15' added to '3.95' is inconsistent with the other buttons.
Docs show these are normal co-ordinate values:
https://github.com/minetest/minetest/blob/bf45644041693f959ac95154b00f15aa27b34145/doc/lua_api.txt#L1962

**Comment by ClobberXD** [comment]
Looks good to me 👍

**Pull request merged**